app/migrations/versions/bb92df9cb6e3_.py

```diff
@@ -0,0 +1,27 @@
+"""empty message
+
+Revision ID: bb92df9cb6e3
+Revises: 2f8c71039480
+Create Date: 2020-10-20 15:42:49.835818
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'bb92df9cb6e3'
+down_revision = '2f8c71039480'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.execute("ALTER TYPE blockchainstatus ADD VALUE 'UNSTARTED'")
```

Comment:
I think we should probably also convert the previous Credit Transfer's `blockchain_status=UNSTARTED` whom have `transfer_status=REJECTED`